Jbarget/oxymore
# adds lazy load to all images

### What changes have you made?

- adds lazy load to all images to improve performance 

### Which issue(s) does this PR fix?

Fixes #389 

### Screenshots (if there are design changes)

### How to test
- Move through the site and ensure that all images load i.e. no non-rendering 
- Remove the `<LazyLoad>` from one or two components and test performance in Lighthouse using a regular `div` 

Before lazy load: 
<img width="1417" alt="Screenshot 2021-02-19 at 19 15 56" src="https://user-images.githubusercontent.com/53219789/108551071-4b397280-72e7-11eb-85ff-60f2653400e7.png">

After: 
<img width="1417" alt="Screenshot 2021-02-19 at 19 16 43" src="https://user-images.githubusercontent.com/53219789/108551055-4674be80-72e7-11eb-9359-5a3d21e662e3.png">
